opencraft/settings.py

```diff
@@ -77,7 +77,8 @@
     'pr_watch',
     'userprofile',
     'registration',
-    'backup_swift'
+    'backup_swift',
+    'community',
```

Comment:
@pomegranited I don't seem to be able to answer to your message below in a thread, so since this is related to this, replying here:

> 

> @antoviaque We could generate the example periodically with a vanilla env file, and just publish that result to our documentation. But if we want this URL to be continually updated, I don't know how to do it better/safer than what @bradenmacdonald proposed.

Well it will be continuously run and updated on the integration tests - maybe we could add a step in the CI build to upload it to a static URL. That would ensure that this is part of the tests, and thus not broken inadvertently, yet still making the result public, and without having to maintain a dedicated service for this.